demo-app/src/app/form-validation/form-validation.component.html

```diff
@@ -0,0 +1,192 @@
+<h3>Form validation</h3>
+
+<div class="section">
+  <p>
+    The <code class="language-markup">mz-validation</code> can be used as a directive on <strong>input</strong>, <strong>input checkbox</strong> and <strong>select</strong>.
+    The component module must use the <a href="https://angular.io/docs/ts/latest/guide/reactive-forms.html" target="_blank">Reactive Form</a> to work with the validation directive.
+  </p>
+
+  <p>
+    <code class="language-markup">ReactiveFormModule</code> should be added in component module or application module.
+  </p>
+  <markdown-to-html [src]="'app/form-validation/markdown/component.module.diff'"></markdown-to-html>
+
+  <p>
+    <strong>Tips 1</strong>: To disable the browser native validation (example: when you click Enter on any required field), you have to add <code class="language-markup">novalidate</code> attribute to form element.
+  </p>
+  <p>
+    <strong>Tips 2</strong>: To make a field required in form we have to add both the <code class="language-markup">required</code> attribute on HTML element and add <code class="language-markup">Validators.required</code> function
+    as validator function in form control contructor. At the moment there is no method in Angular 2 to get the validator function list applied on a from control.
+  </p>
+</div>
+
```

Comment:
Maybe it would be a good idea to add a basic sample of code snippet for one field like the other pages (html, ts) as it is harder to get a quick example without having to understand all the code in the playground